great-idea/css/index.css

```diff
@@ -63,4 +63,251 @@ h1, h2, h3, h4, h5 {
+	background: rgb(245, 243, 243);
+	border: 1px solid gray;
+	border-radius: 2%;
+}
+.thirdLineBlock {
+	display: flex;
+	margin: 40px 0px;
+	padding: 40px 0px 5px 0px;
+	border-top: 1px solid gray;
+}
+.fifthLine {
+	padding: 5px 15px;
+}
+
+.fifthLineBlock {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	margin: 40px 0px;
+}
+footer {
+	border-top: 1px solid gray;
+	padding: 30px 0px;
+	margin: 30px 0px 0px 0px;
+}
+
+.contactInfo li{
+	margin: 8px 0px;
+	list-style-type: none;
+}
+
+.copyright1 {
+	text-align: center;
+	padding: 40px 0px 0px 0px;
+}
+
+/* start of service.css */
+
+
+.header2 {
+	display: flex;
+	justify-content: space-between;
+	margin: 25px 0px;
+	align-items: baseline;
+}
+
+nav {
+	display: flex;
+	justify-content: space-between;
+}
+
+.topSect {
+	display: flex;
+	margin: 20px 0px 15px 0px;
+
+}
+
+.servDes {
+	display:flex;
+	align-items: flex-start;
+	
+}
+
+.firstRow {
+	display: flex;
+	justify-content: space-between;
+	margin: 40px 0px 0px 0px;
+	padding: 30px 0px 0px 0px;
+	border-top: 1px solid gray;
+}
+
+.firstRow a {
+	
+	text-align: center;
+	width: 140px;
+	margin: 15px 0px;
+	padding: 5px 10px;
+	background: rgb(245, 243, 243);
+	border: 1px solid gray;
+	border-radius: 5%;
+
+}
+.design{
+	display: flex;
+	flex-direction: column;
+	margin: 15px;
+	border: 1px solid black;
+	padding: 20px;
+	background: rgb(241, 240, 240);
+}
+.ux {
+	display: flex;
+	flex-direction: column;
+	margin: 15px;
+	border: 1px solid black;
+	padding: 20px;
+	background: rgb(240, 238, 238);
+}
+
+.secondRow {
+	display: flex;
+	justify-content: space-between;
+	margin: 40px 0px 0px 0px;
+}
+.secondRow a {
+	text-align: center;
+	width: 140px;
+	margin: 15px 0px;
+	padding: 5px 10px;
+	background: rgb(245, 243, 243);
+	border: 1px solid gray;
+	border-radius: 5%;
+}
+.digital {
+	display: flex;
+	flex-direction: column;
+	margin: 15px;
+	border: 1px solid black;
+	padding: 20px;
+	background: rgb(240, 238, 238);
+}
+.webDev{
+	display: flex;
+	flex-direction: column;
+	margin: 15px;
+	border: 1px solid black;
+	padding: 20px;
+	background: rgb(240, 238, 238);
+}
+
+.thirdRow {
+	display: flex;
+	justify-content: space-between;
+	margin: 40px 0px 0px 0px;
+}
+.thirdRow a{
+	
+	text-align: center;
+	width: 140px;
+	margin: 15px 0px;
+	padding: 5px 10px;
+	background: rgb(245, 243, 243);
+	border: 1px solid gray;
+	border-radius: 5%;
+}
+.ios {
+	display: flex;
+	flex-direction: column;
+	margin: 15px;
+	border: 1px solid black;
+	padding: 20px;
+	background: rgb(240, 238, 238);
+}
+.android {
+	display: flex;
+	flex-direction: column;
+	margin: 15px;
+	border: 1px solid black;
+	padding: 20px;
+	background: rgb(240, 238, 238);
+}
+
+.footer {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	margin: 80px 0px 0px 0px;
+	padding: 50px 0px 0px 0px;
+	border-top: 2px solid black;
+
+}
+
+.facts {
+	display: flex;
+	flex-direction: column;
+	margin: 15px;
+}
+ul {
+	display: flex;
```

Comment:
You got your unordered list to align correctly!! love the margins! and the use of changing the list style type!!